### PR TITLE
for a review for the first time

### PR DESCRIPTION
**Pull request by Artem18084**
- [DEMO LINK](https://Artem18084.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Demo link and test report links are not working

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_search-bar-airbnb/report/html_report/)

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 700;
+}
+
+/* .styles__form__small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 700;
+} */
```
Comment (VinnieJ-2k20) [inline src/style.css:43]: Don't leave commented code.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Focus styles work incorrectly.
Input should fill the form block completelyfor focus styles to work
image

Also, lots of common styles for big and small elements are used. Put them in common classes for form, input or image.

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Artem18084.github.io/layout_search-bar-airbnb/report/html_report/)

**Review by VinnieJ-2k20 — APPROVED** [review]

**Pull request closed without merging**